src/drawer.multicanvas.js

```diff
@@ -308,12 +308,21 @@ export default class MultiCanvas extends Drawer {
                 let i = first;
 
                 for (i; i < last; i += step) {
-                    const peak =
-                        peaks[Math.floor(i * scale * peakIndexScale)] || 0;
+
```

Comment:
can you remove this empty line, and rename `let i` to something more meaningful?